### PR TITLE
Issue #DP-000 Fix: Show the view content on diksha label when data is ready.

### PR DESCRIPTION
**Pull request by niteshkesarkar**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts [hunk 1]
```diff
@@ -74,6 +74,7 @@ export class QuestionListComponent implements OnInit, AfterViewInit, OnDestroy {
   public sourcingReviewStatus: string;
   public sourcingOrgReviewComments: string;
   originPreviewUrl: string = '';
+  originPreviewReady = false;
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:77]: no need of this originPreviewReady as you are anyway setting it to true if sourcingReviewStatus is Approved

Comment (niteshkesarkar) [reply src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:77]: @twsvaishali  - if we remove `originPreviewReady` then the initially for 1 second you see `ERROR: content is not available on DIKSHA` and later you see the `View this content on DIKSHA`

File: src/app/resourcebundles/data/consumption/en.properties [hunk 2]
```diff
@@ -1355,6 +1355,8 @@ frmelmnts.lbl.contributionRejected=Contributions Rejected
 frmelmnts.lbl.contributionPending=Contributions Pending
 frmelmnts.lbl.addToLibrary= Add to library
 frmelmnts.lbl.viewContent=View Content
+frmelmnts.lbl.contentIsApproved=Content is Approved
```
Comment (twsvaishali) [inline src/app/resourcebundles/data/consumption/en.properties:1358]: As per kamesh's comment, we do not need content is Approved
https://project-sunbird.atlassian.net/browse/DP-743

Comment (twsvaishali) [reply src/app/resourcebundles/data/consumption/en.properties:1358]: @niteshkesarkar

Comment (niteshkesarkar) [reply src/app/resourcebundles/data/consumption/en.properties]: removed related changes to this.

**Pull request merged**